h2o-core/src/main/java/water/parser/ParseSetup.java

```diff
@@ -40,6 +41,9 @@
   String[][] _na_strings;       // Strings for NA in a given column
   String[][] _data;           // First few rows of parsed/tokenized data
   int[] _parse_columns_indices; // store column indices to be parsed into the final file
+  
+  String[] _synthetic_column_names;
```

Comment:
please explain in a comment the purpose of these 2 fileds

ParseSetup is Iced and `_synthetic_column_values` don't seem to be iced-serializable

Reply:
converted to array